### PR TITLE
fix  insert right SQL types values, non stringify

### PR DESCRIPTION
**Pull request by AlvaroVega**
Issue: https://github.com/telefonicaid/fiware-cygnus/issues/1661

This PR would break backward compatibility since a VARCHAR type is common used as type. Do we really want it ?

NGSIType | PostGIS | MySQL | PostgreSQL
------------- | -----------| ---------- | ---------------
Number    |     [X]      |     [¿?]      |         [X]

### CODE REVIEW

**Comment by AlvaroVega** [comment]
iot-cygnus                  | time=2019-06-13T07:14:09.298Z | lvl=DEBUG | corr=141645f8-78a2-4791-a776-15aa324415bf | trans=787262d1-cffa-4173-8913-9b07769fe5a1 | srv=smartcity | subsrv=/ | comp=cygnus-ngsi | op=insertContextData | msg=com.telefonica.iot.cygnus.backends.postgresql.PostgreSQLBackendImpl[192] : Executing SQL query 'INSERT INTO smartcity.thing3_dipsyyy_thing3 (recvTimeTs,recvTime,fiwareServicePath,entityId,entityType,attrName,attrType,attrValue,attrMd) VALUES ('1560410049008','2019-06-13T07:14:09.8Z','/','thing3:dipsYYY','thing3','temperature','Number',**16**,'[]'),('1560410049008','2019-06-13T07:14:09.8Z','/','thing3:dipsYYY','thing3','TimeInstant','DateTime','2019-06-13T07:14:08.00Z','[]')'

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md [hunk 1]
```diff
@@ -119,6 +120,9 @@ Regarding the specific data stored within the above table, if `attr_persistence`
 *  For each notified attribute, a field named as the attribute is considered. This field will store the attribute values along the time.
 *  For each notified attribute, a field named as the concatenation of the attribute name and `_md` is considered. This field will store the attribute's metadata values along the time.
 
+#### <a name="section1.2.6"></a>Native attribute type
+Regarding the specific data stored within the above table, if `attr_native_types` parameter is set to `true` then attribute is inserted using its native type, if `false` then will be strinfy.
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md]: I'd suggest to add the new parameter also to the table:

imagen

Comment (AlvaroVega) [reply doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md]: Where is that table?

Comment (fgalan) [reply doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md]: I think is here: https://github.com/telefonicaid/fiware-cygnus/blob/5e0da0489023e2b866d1d661d2d083bf5f7c07ab/doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md#configuration

Comment (AlvaroVega) [reply doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md]: fixed in 2cdb9f1b32c4567433b800c011250571abf4539b

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md [hunk 2]
```diff
@@ -236,6 +240,7 @@ Coming soon.
 | postgresql\_username | no | postgres | `postgres` is the default username that is created automatically when install |
 | postgresql\_password | no | N/A | Empty value by default (No password is created when install) |
 | attr\_persistence | no | row | <i>row</i> or <i>column</i>. |
+| attr\_native\_types | no | false | <i>false</i> or <i>true</i>. |
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md]: Maybe the description is too "raw". I'd suggest adding an intra-link to the section where you explain this (e.g. `See more detail in [the specific section](#section1.2.6)` or `<a>`, not sure which one will work).

**Pull request merged**